src/kafka-connect/configuration/extraction/configuration-topics.filter.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  'use strict';
+
+  /**
+   * Extracts the type from the connector configuration
+   * @requires supportedConnectorsFactory
+   */
+  angularAPP.filter('configurationType', function (supportedConnectorsFactory) {
```

Comment:
I think names are a bit messed up,
`configuration-topics.filter.js` for `configurationType` filter and visa versa, apart from that, ready to merge 👍